evals/initStagehand.ts

```diff
@@ -0,0 +1,75 @@
+/**
```

Comment:
we have a file called `stagehand.config.ts` in `examples/`. Let's follow that pattern here as well.